lib/passenger.rb

```diff
@@ -16,6 +16,18 @@ def add_trip(trip)
       @trips << trip
     end
 
+    def net_expenditures
+      return nil if @trips.empty?
+
+      return @trips.reduce(0) { |total_cost, trip| total_cost + ( trip.cost || 0 ) } # if trip.cost is falsy, treat as 0
```

Comment:
This can be simplified using a technique similar to what you used in `Driver`'s `average_rating` method.

```suggestion
      return @trips.map(&:cost).compact.sum
```

Note: this version returns 0 if there are no trips, not `nil`.